src/ProjectFileManager.cpp

```diff
@@ -1036,12 +1041,12 @@ void ProjectFileManager::FixTracks(TrackList& tracks,
                RealtimeEffectList::Get(*right).Clear();
 
                if(left->GetRate() != right->GetRate())
-                  //i18n-hint: explains why opened project was auto-modified 
+                  //i18n-hint: explains why opened project was auto-modified
                   onUnlink(XO("This project contained stereo tracks with different sample rates per channel."));
                if(left->GetSampleFormat() != right->GetSampleFormat())
-                  //i18n-hint: explains why opened project was auto-modified  
+                  //i18n-hint: explains why opened project was auto-modified
```

Comment:
This file has too many whitespace changes

Reply:
VSCode does that for me on save. I could only stage and commit the changes that are not automatic, but we're better without those trailing whitespaces anyways.